pkg/resolver/search.go

```diff
@@ -33,20 +34,28 @@ type SearchResult struct {
 	params []interface{}
 	level  int // The number of levels/hops for finding relationships for a particular resource
 	//  Related []SearchRelatedResult
+	userAccess *rbac.UserDataCache
 }
 
 func Search(ctx context.Context, input []*model.SearchInput) ([]*SearchResult, error) {
 	// For each input, create a SearchResult resolver.
 	srchResult := make([]*SearchResult, len(input))
+	userAccess, userDataErr := getUserAccessData(ctx)
+	if userDataErr != nil {
+		return nil, userDataErr
+	}
```

Comment:
1. Let's log this error here. I't not being logged inside the function.
2. I think this error is included in the API query response, so we may need to convert into a more generic error like: "Unable to resolve query because of error while resolving user's access."
3. Maybe we should return the empty `srchResult` instead of nil. Will need to compare with V1.

Reply:
Is it okay to log the error inside the function? So, other resolvers like schema and searchComplete can use it too?